CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 24.11
+
+* When using `sip`, the following endpoints are now making sure the caller's line is registered before starting a call:
+  * `POST /1.0/calls`
+  * `POST /1.0/users/me/calls`
+  In case the requested line is not registered, the error `call-origin-unavailable` is raised
```

Comment:
```suggestion
*  the following endpoints are now performing additional validation before starting a call, which introduce a different behavior in existing error conditions:
  * `POST /1.0/calls`
  * `POST /1.0/users/me/calls`
  In case the requested source line is of type SIP, and is not registered or available, the API will respond with a 402 status with error id `call-origin-unavailable` 
```